docs/UserGuide.md

```diff
@@ -342,27 +384,45 @@ You should use `redo` in conjunction with the [undo](#undoing-a-command--undo) c
 
 Example Command | Result
 --------------- | ----------
-redo | Previously undone command is invoked again
+`redo` | Previously undone command is invoked again
 
 The redo command will not execute if there are no undone commands available to undo. Once _bookmark_ is closed, the
 commands that have been undone during your last usage session cannot be redone using the `redo` command when you
 start _bookmark_ again.
 
-### Clearing all books : `clear` [coming soon]
+### Clearing all books : `clear`
 
-Clears all books from the Storage Library.
+Looking to clear your entire book records? Use the Clear Command to swiftly delete all books at one go!. 
+<div markdown="block" class="alert alert-danger">
+:warning: **Warning: **
+Take caution! While the undo feature will help you restore books deleted by `Clear`, you will not be able
+to recover your deleted books if you call too many commands (10 and more) after clearing!
+</div>
+
+Example Command | Result
+--------------- | ----------
+`clear` | Clears the entire _bookmark_ storage
 
 Format: `clear`
 
 ### Exiting the program : `exit`
 
-Exits the program.
+Ready to continue your reading? Close the program by using our Exit Command to safely exit the 
+program. 
 
 Format: `exit`
 
+Example Command | Result
+---------------- | ---------
+`exit` | Exits _bookmark_
+
 ### Saving the data
 
 _bookmark_ data will be saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+<div markdown="block" class="alert alert-warning">
+:star: **Important Detail:**
+Do take note in exiting _bookmark_ properly to ensure that your records are stored!
```

Comment:
Not too sure if we should mention this as we mention "There is no need to save manually", but then we tell users that they need to exit properly